### DISCUSSION TITLE
Unit tests with Transloco

### DISCUSSION THREAD

**Discussion by cguilhermef** [ngneat/transloco#413] [Q&A]
Hi there!

Is mandatory to use the @ngneat/spectator to achieve unit tests of components? 
In our project, when using directive, our tests are failing. But, when we switch to use the pipe alternative, eventually works. Why?
What are the common practices? Why the documentation doesn't do any mention on it?

**Comment by shaharkazaz** [comment]
@cguilhermef you don't have to use [@ngneat/spectator](https://github.com/ngneat/spectator) in order to test your components.
I don't know what's your specific scenario is, can you provide a reproduction?
